src/Stripe.net/Constants/Events.cs

```diff
@@ -1,8 +1,6 @@
 // File generated from our OpenAPI spec
 namespace Stripe
 {
-    using System;
```

Comment:
Why was this removed? Did this do anything before?

Reply:
The `Obsolete` tag on the events that got removed need `System`